aspnetcore/fundamentals/http-requests.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Making HTTP Requests
+author: stevejgordon
+description: Learn about using the HttpClientFactory features to 
+manager: wpickett
+ms.author: riande
+ms.date: 02/15/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: fundamentals/http-requests
+---
+# Making HTTP Requests
+
+By [Glenn Condron](https://github.com/glennc), [Ryan Nowak](https://github.com/rynowak) and [Steve Gordon](https://github.com/stevejgordon) 
+
+A `HttpClientFactory` can be registered and used to configure and consume `HttpClient` instances in your application. It provides several benefits:
+
+1. Provides a central location for naming and configuring logical HttpClients. For example, you may configure a “github” client that is pre-configured to access github and a default client for other purposes.
+2. Codifies the concept of outgoing middleware via delegating handlers in HttpClient and implementing Polly based middleware to take advantage of that. HttpClient already has the concept of delegating handlers that could be linked together for outgoing HTTP requests. The factory will make registering of these per named client more intuitive as well as implement a Polly handler that allows Polly policies to be used for Retry, CircuitBreakers, etc.
+3. Manage the lifetime of HttpClientMessageHandlers to avoid common DNS problems that can be hit when managing HttpClient lifetimes yourself.
+
+## Consumption patterns
+
+There are several ways that `HttpClientFactory` can be used in your application. None of them are strictly superior to another, it really depends on your application and the constraints you are working under.
+
+## Basic usage
+
+The `HttpClientFactory` can be used directly in your code to access `HttpClient` instances. First the services must be registered with the ServiceProvider.
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddHttpClient();
+    services.AddMvc();
+}
+```
+
+Once registered, you can accept a `IHttpClientFactory` in your constructor which can then be used to create a `HttpClient`.
```

Comment:
Probably want to generalize this a bit, make sure people know it is DI magic and can work wherever you can inject services. Not just constructors.